### PR TITLE
reorg README and add ToC

### PR DESCRIPTION
**Pull request by jnny**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,214 +55,42 @@ on AMD64 platforms.  Building and testing Ouinet on your computer requires
...
-case of a crash.  Just add `--build-arg OUINET_DEBUG=yes` to the build
-command.  We recommend that you use a different tag for these images
-(e.g. `equalitie/ouinet:<VERSION>-debug`).
-
-Depending on your Docker setup, you may need to change the container's
-security profile and give it tracing capabilities.  For more information, see
-[this thread](https://stackoverflow.com/q/35860527).
-
-### Deploying a client
+Table of Contents
```
Comment (ivilata) [inline README.md:58]: Please use `#` header for consistency with the rest of the doc.

File: README.md [hunk 2]
```diff
@@ -55,214 +55,42 @@ on AMD64 platforms.  Building and testing Ouinet on your computer requires
...
+- [Setup a Development Environment](#development-environment)
+    - [Building from Source](#building-from-source)
+    - [Docker Development Environment](#docker-development-environment)
+    - [Vagrant Development Environment](#vagrant-development-environment)
+
+## Deploy a client or injector with Docker 
+
+Ouinet injectors and clients can be run as Docker containers.  Ouinet Docker images are available on the [Docker Hub](https://hub.docker.com/equalitie/ouinet).  An application configuration file for Docker Compose is included for easily deploying all needed volumes and containers.
+
+To run a Ouinet node container only a couple hundred MiB are needed, plus the space devoted to the data volume (which may grow considerably larger in the case of the injector).
```
Comment (ivilata) [inline README.md:83]: Injectors don't cache content, so they need no extra space after first start.

File: README.md [hunk 3]
```diff
@@ -55,214 +55,42 @@ on AMD64 platforms.  Building and testing Ouinet on your computer requires
...
+    - [Running a Test Client](#running-a-test-client)
+    - [Testing with a Browser](#testing-the-client-with-a-browser)
+- [Setup a Development Environment](#development-environment)
+    - [Building from Source](#building-from-source)
+    - [Docker Development Environment](#docker-development-environment)
+    - [Vagrant Development Environment](#vagrant-development-environment)
+
+## Deploy a client or injector with Docker 
+
+Ouinet injectors and clients can be run as Docker containers.  Ouinet Docker images are available on the [Docker Hub](https://hub.docker.com/equalitie/ouinet).  An application configuration file for Docker Compose is included for easily deploying all needed volumes and containers.
```
Comment (ivilata) [inline README.md:81]: URL not working (yet?).

**Review by jnny — COMMENTED** [review]
reviewed & fixed comments from @ivilata

**Pull request closed without merging**